src/engine/engine.cpp

```diff
@@ -491,6 +491,72 @@ void DivEngine::notifyWaveChange(int wave) {
   BUSY_END;
 }
 
+unsigned char* loadSampleRom(String path, ssize_t expectedSize) {
+  if (path.empty()) {
+    return NULL;
+  }
+  logI("loading ROM %s...",path);
+  FILE* f=ps_fopen(path.c_str(),"rb");
+  if (f==NULL) {
+    perror("error");
```

Comment:
Replace instances of perror() with logW() or logE()? (use strerror(errno) to get last error message)

Reply:
Better like this?

p.s. I copy/pasted the perrors from elsewhere in the source.

Reply:
All good now. Yep, I know...